packages/remix-server-runtime/routeModules.ts

```diff
@@ -66,6 +78,8 @@ export interface LinksFunction {
 
 /**
  * A function that loads data for a route.
+ *
+ * @deprecated in favor of React Router `LoaderFunction`
```

Comment:
We might not be able to include this deprecation yet since the naming is the same and we don't want folks to start importing from `react-router-dom`?